docs/IERC777Recipient.md

```diff
@@ -0,0 +1,26 @@
+# [🔗](/interfaces/IERC777Recipient.sol#L7) IERC777Recipient
+**Author** _Andrea Di Nenno_
+
+This interface allows a token receiver (EOA or Smart Contract) to add extra logic to control its incoming tokens.
+
+
+# Functions
+## [🔗](/interfaces/IERC777Recipient.sol#L16) `tokensReceived(address operator, address from, address to, uint256 amount, bytes userData, bytes operatorData)`
+
+Called by an `ERC777` token contract whenever tokens are being moved or created into a account.

```

Comment:
Sorry missed the -ing and a-an

Called by an `ERC777` token contract whenever tokens are moved or created into an account.